libplatsupport/src/mach/bcm/pl011_uart.c

```diff
@@ -227,14 +230,14 @@ static int pl011_uart_configure(ps_chardevice_t *dev)
      *
      */
     // Enable FIFO
-    //pl011_uart_enable_fifo(dev);
+    pl011_uart_enable_fifo(dev);
```

Comment:
I'm not convinced that the problem mentioned in the comment is properly addressed. I am afraid that enabling FIFOs is just a work-around for an unfixed bug, probably caused by the original's code assumptions of the current state of the UART or initialisation.

Have you tried fully configuring the UART and setting all registers, and clearing any pending IRQs and errors at init?

Reply:
I tested the current state again today and it is working fine.

Sending single bytes is not an issue. 
For testing I did setup a simple uart echo server which is interrupt triggered.

![2024-03-15_13-43](https://github.com/seL4/util_libs/assets/82309717/29afc2dc-1313-409c-8b1f-8aa6284fc018)

Reply:
Please print something out at every receive interrupt and check that you don't get any when no input is given.

Or keep a counter and print out the counter value too.